codegen/gateway_exec.go

```diff
@@ -33,8 +33,9 @@ func ExecWithFS(ctx context.Context, cln *client.Client, fs Filesystem, opts Opt
 		cwd = fs.Image.Config.WorkingDir
 	}
 
-	env := make([]string, len(fs.Image.Config.Env))
+	env := make([]string, len(fs.Image.Config.Env)+len(extraEnv))
```

Comment:
This doesn't look right if you're using `append` below. You would want:

```go
env := make([]string, len(fs.Image.Config.Env), len(fs.Image.Config.Env)+len(extraEnv))
```

Reply:
Oops indeed! Fixed now.